cs/src/comm/interfaces/Message.cs

```diff
@@ -201,46 +197,36 @@ public static IMessage<TPayload> FromError<TPayload>(IBonded<Error> err)
         }
 
         /// <summary>
-        /// Creates an error message from the given error.
+        /// Creates an Error message from the given error.
```

Comment:
You may want to visit the casing in this whole file. To me, the following is more accurate:
Creates an error message from the given Error.